docs/blog/newsletter-october-2017.rst

```diff
@@ -0,0 +1,81 @@
+*************************************************
+How do we make our URLs meaningful to our readers? What should we do when content inevitably moves (or is removed)? How can we make sure all of the great content we write remains discoverable?
+
+When discussing these questions this month, the community started from a point of agreement: good URL paths should carry meaning. For example, a lot of doc sites have URLs that reflect their content hierarchy, like ``site/category/article-title``. Ideally, in such situations, if a reader truncates the path to ``site/category/`` they should find a list of every article in the category.But content restructuring and obsolescence are facts of life for most documentarians. It's unrealistic to think that `Cool URIs don't change <https://www.w3.org/Provider/Style/URI>`_. When the inevitable does happen, it's important to make it as easy as possible for readers to find new locations. That means redirects!
+
+Creating redirects can be difficult to manage when there are lots of external links when lots of URLs are changing. Sadly, most documentation tools – with the notable exception of MindTouch – don't offer automated redirects. One suggested approach was to use analytics to prioritize your most critical URLs, setting up manual redirects for those pages first to keep your content discoverable. For long term URL hygiene, folks suggested automated link-checking tools to find track down and update broken links and stray pages.
+
+*************************
+The true meaning of "API"
+*************************
+What exactly are we referring to when we say "API"? While the initialism itself stands for Application Programming Interface, it's a more complicated question than it seems. When it came up this month, it sparked a lively conversation that suggested that API might mean different things to different people in different contexts.
+
+Most of the conversation revolved around that idea that, when we say "API" we assume that our reader (or listener) will inherently understand the type of API we're talking about. But that might not be the case as often as you think. If you work with REST APIs, it's likely you'll assume that API has a silent "REST" in front of it. But if you've spent your career working with library APIs, that will probably be the meaning you default to.
+
+There was some question of whether any of this mattered. Whether it's a library, REST, SOAP, or other type of API, they're all still "methods of communication between various software components" (thank you `wikipedia <https://en.wikipedia.org/wiki/Application_programming_interface>`_). But as evidenced by the length and depth of the conversation, it's definitely a topic worth clarifying. Handily enough, one of our Prague speakers, Garen Torikian discusses has a slide on this very topic in his presentation, "Finally! Trustworthy and Sensible API Documentation with GraphQL." Stay tuned for the video recording!
```

Comment:
"Garen Torikian discusses has a slide on this very topic" - We should strike "discusses".

Reply:
We should also have a comma after Garen Torikian.

Reply:
👍